snap/implicit_test.go

```diff
@@ -56,7 +56,7 @@ func (s *InfoSnapYamlTestSuite) TestAddImplicitSlotsOnClassic(c *C) {
 	c.Assert(info.Slots["unity7"].Interface, Equals, "unity7")
 	c.Assert(info.Slots["unity7"].Name, Equals, "unity7")
 	c.Assert(info.Slots["unity7"].Snap, Equals, info)
-	c.Assert(info.Slots, HasLen, 17)
+	c.Assert(info.Slots, HasLen, 18)
```

Comment:
Can this be changed to say "more than 1" instead.

```c.Assert(len(info.Slots) > 1, Equals, true)```

Each new interface has to touch this.